docs/tutorials/flow-app-quickstart.mdx

```diff
@@ -163,441 +153,14 @@ export default function Home() {
-        }
-      `,
-      args: (arg, t) => [arg(user.addr, t.Address)]
-    })
-
-    setName(profile?.name ?? 'No Profile')
-  }
-
-  const initAccount = async () => {
-    const transactionId = await fcl.mutate({
-      cadence: `
-        import Profile from 0xProfile
-
-        transaction {
-          prepare(account: AuthAccount) {
-            // Only initialize the account if it hasn't already been initialized
-            if (!Profile.check(account.address)) {
-              // This creates and stores the profile in the user's account
-              account.save(<- Profile.new(), to: Profile.privatePath)
-
-              // This creates the public capability that lets applications read the profile's info
-              account.link<&Profile.Base{Profile.Public}>(Profile.publicPath, target: Profile.privatePath)
-            }
-          }
-        }
-      `,
-      payer: fcl.authz,
-      proposer: fcl.authz,
-      authorizations: [fcl.authz],
-      limit: 50
-    })
-
-    const transaction = await fcl.tx(transactionId).onceSealed()
-    console.log(transaction)
-  }
-
-  // NEW
-  const executeTransaction = async () => {
-    const transactionId = await fcl.mutate({
-      cadence: `
-        import Profile from 0xProfile
-
-        transaction(name: String) {
-          prepare(account: AuthAccount) {
-            account
-              .borrow<&Profile.Base{Profile.Owner}>(from: Profile.privatePath)!
-              .setName(name)
-          }
-        }
-      `,
-      args: (arg, t) => [arg("Flow Developer!", t.String)],
-      payer: fcl.authz,
-      proposer: fcl.authz,
-      authorizations: [fcl.authz],
-      limit: 50
-    })
-
-    fcl.tx(transactionId).subscribe(res => setTransactionStatus(res.status))
-  }
-
-  const AuthedState = () => {
-    return (
-      <div>
-        <div>Address: {user?.addr ?? "No Address"}</div>
-        <div>Profile Name: {name ?? "--"}</div>
-        <div>Transaction Status: {transactionStatus ?? "--"}</div> {/* NEW */}
-        <button onClick={sendQuery}>Send Query</button>
-        <button onClick={initAccount}>Init Account</button>
-        <button onClick={executeTransaction}>Execute Transaction</button> {/* NEW */}
-        <button onClick={fcl.unauthenticate}>Log Out</button>
-      </div>
-    )
-  }
-
-  const UnauthenticatedState = () => {
-    return (
-      <div>
-        <button onClick={fcl.logIn}>Log In</button>
-        <button onClick={fcl.signUp}>Sign Up</button>
-      </div>
-    )
-  }
+You should now be able to log in or sign up a user and unauthenticate them. Upon logging in or signing up, your users will see a popup where they can choose between wallet providers. You will select the [Blocto wallet](https://blocto.portto.io/) for this example to create an account. Upon completing authentication, you will see the component change, and the user's wallet address appears on the screen. You should see a similar image below if you have completed this properly.
```

Comment:
> we should also mention lilico is available, even if we don't use it, to truly show the power of FCL

When I was testing it with my PC, lilico didn't show and it might be the same with other developers using it for the first time.